src/SDKs/SqlManagement/Sql.Tests/DatabaseRestoreScenarioTests.cs

```diff
@@ -163,104 +163,138 @@ public void TestDatabaseRestore()
-                Assert.Equal(vault.Id, serverVault.RecoveryServicesVaultResourceId, ignoreCase: true);
-
-                // List server LTR backup vaults
-                IEnumerable<BackupLongTermRetentionVault> serverVaults = sqlClient.BackupLongTermRetentionVaults.ListByServer(resourceGroup.Name, server.Name);
-                Assert.Single(serverVaults);
-                Assert.Equal(vault.Id, serverVaults.Single().RecoveryServicesVaultResourceId, ignoreCase: true);
-
-                // Create database LTR policy
-                Database db = sqlClient.Databases.CreateOrUpdate(
-                    resourceGroup.Name, 
-                    server.Name,
-                    databaseName: SqlManagementTestUtilities.GenerateName(), 
-                    parameters: new Database(resourceGroup.Location));
-                BackupLongTermRetentionPolicy databasePolicy = sqlClient.BackupLongTermRetentionPolicies.CreateOrUpdate(
-                    resourceGroup.Name, server.Name, db.Name, new BackupLongTermRetentionPolicy(
-                        BackupLongTermRetentionPolicyState.Enabled,
-                        policy.Id));
-
-                // Get database LTR policy
-                databasePolicy = sqlClient.BackupLongTermRetentionPolicies.Get(resourceGroup.Name, server.Name, db.Name);
-                Assert.Equal(policy.Id, databasePolicy.RecoveryServicesBackupPolicyResourceId, ignoreCase: true);
-
-                // List database LTR policies
-                IEnumerable<BackupLongTermRetentionPolicy> databasePolicies = sqlClient.BackupLongTermRetentionPolicies.ListByDatabase(resourceGroup.Name, server.Name, db.Name);
-                Assert.Single(databasePolicies);
-                Assert.Equal(policy.Id, databasePolicies.Single().RecoveryServicesBackupPolicyResourceId, ignoreCase: true);
-
-                // Update database LTR policy
-                databasePolicy = sqlClient.BackupLongTermRetentionPolicies.CreateOrUpdate(
-                    resourceGroup.Name, server.Name, db.Name, new BackupLongTermRetentionPolicy(
-                        BackupLongTermRetentionPolicyState.Disabled,
-                        policy.Id /* policy Id must be set even when disabling */));
-
-                // Restore from LTR
-                // Commented out because there can be a delay of several hours before the first
-                // LTR backup is created. This code is just to show example.
-                /*
-                Database restoredDatabase = sqlClient.Databases.CreateOrUpdate(
-                    resourceGroup.Name, server.Name, databaseName: SqlManagementTestUtilities.GenerateName(),
+        [Fact(Skip = "Manual test due to long setup time required (over 18 hours).")]
+        public void TestLongTermRetentionV2Crud()
+        {
+            string locationName = "westcentralus";
+            string resourceGroupName = "Default-SQL-WestCentralUs";
+            string serverName = "trgrie-ltr-server";
```

Comment:
depersonalize server name and add manual setup instructions (i.e. "please create a server & database and then wait x hours and fill in the names here")